addons/website_sale/security/website_sale.xml

```diff
@@ -12,20 +12,25 @@
         <field name="perm_unlink" eval="False"/>
     </record>
 
+    <record id="group_ecom_uom_price" model="res.groups">
+        <field name="name">UOM Price Display for eCommerce</field>
+        <field name="category_id" ref="base.module_category_hidden"/>
+    </record>
+
     <record id="sales_team.group_sale_manager" model="res.groups">
         <field name="implied_ids" eval="[(4, ref('website.group_website_publisher'))]"/>
     </record>
 
     <record id="base.group_user" model="res.groups">
-        <field name="implied_ids" eval="[(4, ref('sale.group_delivery_invoice_address'))]"/>
+        <field name="implied_ids" eval="[(4, ref('sale.group_delivery_invoice_address')), (4, ref('website_sale.group_ecom_uom_price'))]"/>
     </record>
 
     <record id="base.group_public" model="res.groups">
-        <field name="implied_ids" eval="[(4, ref('sale.group_delivery_invoice_address'))]"/>
+        <field name="implied_ids" eval="[(4, ref('sale.group_delivery_invoice_address')), (4, ref('website_sale.group_ecom_uom_price'))]"/>
     </record>
 
     <record id="base.group_portal" model="res.groups">
-        <field name="implied_ids" eval="[(4, ref('sale.group_delivery_invoice_address'))]"/>
+        <field name="implied_ids" eval="[(4, ref('sale.group_delivery_invoice_address')), (4, ref('website_sale.group_ecom_uom_price'))]"/>
```

Comment:
You don't want the option checked by default, so it seems useless for 3 occurences